cfgov/jinja2/v1/youth_employment_success/goals.html

```diff
@@ -1,5 +1,5 @@
 {% from "goals/goal.html" import renderTextarea with context %}
-{% from 'atoms/radio-button.html' import render with context %}
+{% from 'atoms/radio-button.html' import render as renderRadio with context %}
```

Comment:
There's also `{% import 'atoms/radio-button.html' as radio_button with context %}` and then using `radio_button.render(…)` where it's needed, which feels more OOP to me. We use both formats though, so just pointing out the alternative.

Reply:
I like that syntax more, ill switch to it. Thanks!